go.mod

```diff
@@ -245,4 +245,6 @@ replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-d
 // automatically, so we need to add it manually.
 replace github.com/golang-migrate/migrate/v4 => github.com/lightninglabs/migrate/v4 v4.18.2-9023d66a-fork-pr-2
 
-go 1.23.6
+replace github.com/lightningnetwork/lnd => github.com/lightningnetwork/lnd v0.19.0-beta
```

Comment:
Could add this replace for the `litrpc` package as well :).

Reply:
It won't do anything for us. Since replaces aren't transitive (meaning it wouldn't affect any consumers of the module) and the `litrpc` itself doesn't need the latest version by itself. But I'll fix this for the final release anyway (by bumping the lnd version in each project).

Reply:
Ah good point! Makes sense, thanks :pray:!